src/blueprint/azext_blueprint/_validators.py

```diff
@@ -20,3 +20,8 @@ def blueprint_assignment_validator(cmd, namespace):
     if namespace.management_group:
         raise CLIError("The management group scope for blueprint assignment is not supported yet. Please use --subscription for subscription scope.")
     namespace.scope = '/subscriptions/{}'.format(namespace.subscription if namespace.subscription else get_subscription_id(cmd.cli_ctx))
+    try:
+        if namespace.user_assigned_identity and not namespace.identity_type:
+            namespace.identity_type = 'UserAssigned'
+    except AttributeError:
+        pass

```

Comment:
Add a warning?

Reply:
The validator is in the scope of `blueprint assignment` while the attribute of `identity_type` only exists in `create/update`. So this error is expected for commands like `show`.